src/dialogs/generic/dialog-box.ts

```diff
@@ -167,6 +170,12 @@ class DialogBox extends LitElement {
     this._closeResolve = undefined;
   }
 
+  protected supportedSingleKeyShortcuts(): SupportedShortcuts {
+    return {
+      Enter: () => this._confirm(),
```

Comment:
This should not be needed and be the default? Also what happens if the focus is on cancel and I press enter?